bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -172,6 +200,10 @@
 		<item-type>Switch</item-type>
 		<label>Super Cool</label>
 		<description>Start or stop Super Cooling</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
I believe we settled on:
```suggestion
			<tag>Enabled</tag>
```

like for the channel type below. See https://github.com/openhab/openhab-addons/pull/18585#discussion_r2059915444